app/routes/docs.notices.schema.mdx

```diff
@@ -65,8 +81,13 @@ This allows our validator script to identify which schema examples to compare. M
       code={JSON.stringify(
         {
           $schema:
-            'https://gcn.nasa.gov/schema/main/gcn/notices/<mission>/<schema name>.schema.json',
-          alert_datetime: 'yyyy-mm-ddThh:mm:ssZ',
+            'https://gcn.nasa.gov/schema/main/gcn/notices/mission/Sample.schema.json',
+          alert_datetime: '2023-09-28T01:40:30Z',
+          ra: 197.44871198,
+          dec: -23.38397612,
+          example_field_1:
+            'The ultimate answer to life, the universe, and everything',
```

Comment:
'The ultimate answer to life, the universe, and everything' 
Sure, ROFL!

Reply:
It's a reference to [The Hitchhiker's Guide to the Galaxy](https://en.wikipedia.org/wiki/The_Hitchhiker%27s_Guide_to_the_Galaxy), as is the number 42.

Reply:
42 number!